tests/bug_1834974.py

```diff
@@ -0,0 +1,18 @@
+import logging
+import time
+logger = logging.getLogger(__name__)
+log = logger
+
+
+def run(**kw):
+    log.info("Bug 1834974")
+    ceph_nodes = kw.get('ceph_nodes')
+    time.sleep(180)
+    for cnode in ceph_nodes:
+        out, err = cnode.exec_command(cmd='sudo df -h | grep -v shm | grep -i containers | wc -l', long_running=True)
```

Comment:
I would have still loved to see if directories exist and not just the size of the container.
Can you look into it? Besides that, the change looks good

Reply:
sorry. I see the checks for containers directory.